google/cloud/bigquery/client.py

```diff
@@ -2562,6 +2576,13 @@ def load_table_from_dataframe(
         if job_config.source_format is None:
             # default value
             job_config.source_format = job.SourceFormat.PARQUET
+
+        if job_config.parquet_options is None:
+            parquet_options = ParquetOptions()
+            # default value
+            parquet_options.enable_list_inference = True
```

Comment:
Does this change the behavior with non-compliant parquet files? Should we add a `pyarrow` version check here too?

That could be nice in that folks can drop to pyarrow 3.0.0 if they really want the old behavior.

Reply:
Yes it does. Goes from `[{list: {item: value}}]` to `[{item: value}]` I think.

Reply:
They could always drop to pyarrow 3.0.0 and set `enable_list_inference` to `False` 😛